app-backend/tile/src/main/scala/Main.scala

```diff
@@ -19,7 +20,15 @@ object AkkaSystem {
 object Main extends App with Config with AkkaSystem.LoggerExecutor {
   import AkkaSystem._
 
-  def rootRoute = pathPrefix("tiles") { Routes.singleLayer }
+  def rootRoute = pathPrefix("tiles") {
+    Routes.singleLayer ~ path("healthcheck" /) {
```

Comment:
Not sure exactly the way the `path("healthcheck" /)` would behave. Spray is notoriously finicky when optional end slashes are concerned, I might suggest following this pattern from the `app` project:

```scala
pathPrefix("healthcheck") {
  pathEndOrSingleSlash {
    get {
      complete {
        HttpResponse(StatusCodes.OK)
      }
    }
  }
}
```